docs/topology.md

```diff
@@ -0,0 +1,64 @@
+Dependency Topology
+-------------------
+
+# Dependency Resolution
+
+To enable dynamic dependency resolution the following annotations will be added to the `Chart.yaml` files of TSSC Helm charts. These annotations guide the installer in building the correct deployment order, and the namespaces is determined by the installer [configuration](../README.md#-installerproducts-).
```

Comment:
'the following annotations will be added to the `Chart.yaml` ' -> 'annotations are added to the `Chart.yaml`' ?

Reply:
Indeed, thanks for the suggestion.